azure_monitor/setup.cfg

```diff
@@ -1,25 +1,11 @@
-# Copyright 2019, OpenTelemetry Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
 [metadata]
-name = opentelemetry-ext-azure-monitor
+name = opentelemetry-azure-monitor-exporter
 description = Azure Monitor integration for OpenTelemetry
 long_description = file: README.rst
 long_description_content_type = text/x-rst
-author = OpenTelemetry Authors
-author_email = cncf-opentelemetry-contributors@lists.cncf.io
-url = https://github.com/open-telemetry/opentelemetry-python/ext/opentelemetry-ext-azure-monitor
+author = Microsoft
+author_email = appinsightssdk@microsoft.com
+url = https://github.com/microsoft/opentelemetry-exporters-python
 platforms = any
 license = Apache-2.0
```

Comment:
We use MIT license for other packages, do we need to use same license as OpenTelemetry here?

Reply:
http://go.microsoft.com/fwlink/?LinkID=510709

Reply:
Good question. I think @reyang mentioned we should keep the license from OpenTelemetry.
https://github.com/microsoft/opentelemetry-exporters-python/pull/1#pullrequestreview-305578045

I'm not sure what the repercussions of not including the MIT license. Can we have multiple?